tests/fake_messages/platformci_queued_message.json

```diff
@@ -0,0 +1,42 @@
+{
+  "timestamp": 1536855059.0, 
+  "msg_id": "ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-38241-1536153362773-4:537927:0:0:1", 
+  "topic": "/topic/VirtualTopic.eng.ci.brew-build.test.queued", 
+  "headers": {
+    "expires": "0", 
+    "priority": "4", 
+    "message-id": "ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-38241-1536153362773-4:537927:0:0:1", 
```

Comment:
Obfuscate the `message-id` (use similar to one in other test messages).

Reply:
I'm changing it (I'm going to commit the change now), but this is not obfuscated in all of the other examples... so maybe we should change also them in the future in another PR.